Makefile.dep

```diff
@@ -211,7 +211,7 @@ ifneq (,$(filter ipv6_ext_rh,$(USEMODULE)))
 endif
 
 ifneq (,$(filter gnrc_ipv6_ext,$(USEMODULE)))
-  USEMODULE += ipv6_ext
+  # USEMODULE += ipv6_ext
```

Comment:
Please just remove this line or introduce `ipv6_ext` as a pseudo module. I'm open to both :-).